test/functions/compilation-errors/recursive-generic-return.good

```diff
@@ -0,0 +1,2 @@
+recursive-generic-return.chpl:12: warning: RR(int(64))
+recursive-generic-return.chpl:15: error: generic return type 'RR'
```

Comment:
If you make my suggested change to the error message above this line will need to change too.